homeassistant/components/unifi/hub/entity_loader.py

```diff
@@ -43,6 +56,16 @@ def __init__(self, hub: UnifiHub) -> None:
         self.known_objects: set[tuple[str, str]] = set()
         """Tuples of entity description key and object ID of loaded entities."""
 
+    async def refresh_api_data(self) -> None:
+        """Refresh API data from network application."""
+        results = await asyncio.gather(
+            *[update() for update in self.api_updaters],
+            return_exceptions=True,
```

Comment:
I'd check if the gather is slower or faster since sometimes gather can be a lot slower if keep alive is available and gather causes a lot of new connections

Reply:
Did a measurement

sequentially start 10:24:41.796
sequentially stop 10:24:41:909
gather start 10:25:49.005
gather stop 10:25:49.080

sequentially timed 113
gather timed 75